surface/include/pcl/surface/3rdparty/poisson4/multi_grid_octree_data.hpp

```diff
@@ -206,24 +207,36 @@ namespace pcl
       cData.cCount = 0;
       std::vector< int > offsets( threads+1 );
       offsets[0] = 0;
-      for( int t=0 ; t<threads ; t++ ) cData.cCount += count[t] , offsets[t+1] = offsets[t] + count[t];
+      for (int t = 0; t < threads; t++)
+      {
+        cData.cCount += count[t];
+        offsets[t + 1] = offsets[t] + count[t];
+      }
+
+      int paralellExceptionCount = 0;
 #pragma omp parallel for num_threads( threads )
-      for( int t=0 ; t<threads ; t++ )
-        for( int d=minDepth ; d<=maxDepth ; d++ )
+      for (int t = 0; t < threads; t++)
+      {
```

Comment:
Added opening/closing braces for omp `for` loops to help identify end of the OMP scope